src/lib/containers/SearchV2.tsx

```diff
@@ -221,56 +246,56 @@ class Search extends React.Component<InternalSearchProps, SearchState> {
-          classNames="SearchV2__animate_bar"
-          timeout={{ enter: 0, exit: 300 }}
-        >
-          <div className="SearchV2__animate_height">
-            <form
-              className="SearchV2__searchbar"
-              onSubmit={this.search}
+      <div
+        className={`SearchV2 ${
+          showFacetFilter
+            ? QUERY_FILTERS_EXPANDED_CSS
+            : QUERY_FILTERS_COLLAPSED_CSS
+        }`}
+      >
+        <Collapse in={topLevelControlsState?.showSearchBar}>
+          <form
+            className="SearchV2__searchbar"
+            onSubmit={this.search}
+            onClick={() => {
+              this.setState({ show: true })
+            }}
+            ref={this.searchFormRef}
+          >
+            <FontAwesomeIcon
+              className="SearchV2__searchbar__searchicon"
+              size={'sm'}
+              icon={'search'}
+            />
+            <input
+              onChange={this.handleChange}
               onClick={() => {
-                this.setState({ show: true })
+                this.setState({
+                  show: true,
+                })
               }}
-              ref={this.searchFormRef}
-            >
-              <FontAwesomeIcon
-                className="SearchV2__searchbar__searchicon"
-                size={'sm'}
-                icon={'search'}
-              />
-              <input
-                onChange={this.handleChange}
+              placeholder="Enter Search Terms"
+              value={searchText}
+              type="text"
+            />
+            {this.state.searchText.length > 0 && (
+              <button
+                className="SearchV2__searchbar__clearbutton"
+                type="button"
                 onClick={() => {
                   this.setState({
-                    show: true,
+                    searchText: '',
                   })
                 }}
-                placeholder="Enter Search Terms"
-                value={searchText}
-                type="text"
-              />
-              {this.state.searchText.length > 0 && (
-                <button
-                  className="SearchV2__searchbar__clearbutton"
-                  type="button"
-                  onClick={() => {
-                    this.setState({
-                      searchText: '',
-                    })
-                  }}
-                >
-                  <FontAwesomeIcon
-                    className="SRC-primary-text-color"
-                    icon="times"
-                  />
-                </button>
-              )}
-            </form>
-          </div>
-        </CSSTransition>
+              >
+                <FontAwesomeIcon
+                  className="SRC-primary-text-color"
+                  icon="times"
+                />
+              </button>
+            )}
+          </form>
+        </Collapse>
         <div className="SearchV2__dropdown_pos">
           <CSSTransition
```

Comment:
Not using MUI Collapse here because the dropdown is absolutely-positioned, so Collapse doesn't work